<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBar</class>
 <widget class="QWidget" name="ToolBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>87</width>
    <height>214</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0">
    <widget class="QPushButton" name="colorSelection">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="penButton">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="mirrorHorizontal">
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="animationPlay">
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="mirrorVerticle">
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="eraserSizeSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>32</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="penSizeSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>32</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="colorInvertButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="eraserButton">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
